ChemistryLib/PhreeqcIOData/CreateOutput.cpp

```diff
@@ -48,8 +50,16 @@ std::unique_ptr<Output> createOutput(
                                     kinetic_reactant.item_type);
     }
 
+    if (user_punch)
+    {
+        auto const& secondary_variables = user_punch->secondary_variables;
+        std::transform(secondary_variables.begin(), secondary_variables.end(),
+                       std::back_inserter(accepted_items), accepted_item);
```

Comment:
call to `accepted_items.reserve(accepted_items.size() + secondary_variables.size())` would save some reallocations.

Reply:
Added.